templates/src/config/mysql.js

```diff
@@ -0,0 +1,23 @@
+const Sequelize = require('sequelize');
+
+const sequelizeDB = new Sequelize(
+  process.env.DB_NAME,
+  process.env.DB_USERNAME,
+  process.env.DB_PASSWORD, {
+    host: process.env.HOST,
+    port: process.env.PORT,
+    dialect: 'mysql',
+    define: {
+      timestamps: false,
+    },
+    native: false,
+    logging: false,
+    timezone: '+07:00',
+  },
+);
```

Comment:
should use connection string `const sequelize = new Sequelize("mysql://${process.env.MYSQL_URL}");`.
example: `MYSQL_URL=user:pass@example.com:3306/dbname`